.github/workflows/build.yml

```diff
@@ -78,7 +78,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.8, 3.9, "3.10"]
+        python-version: [3.8, 3.9, "3.10", "3.11"]
```

Comment:
Out of curiosity - are we interested in 3.10 here? Shouldn't we ditch it simply? :thinking:

Reply:
I'd put in 3.12 instead, and maybe also the 3.13 alpha, because with the gh action to setup python that's easy thing to do.

This way ci can warn about upcoming problems with new python version.